### ISSUE TITLE
DNS: Multi txt records can not be used

### ISSUE THREAD

**Issue by Slm0n87** [stackitcloud/terraform-provider-stackit#326]
I try to set a multi-record TXT record:

```
resource "stackit_dns_zone" "example" {
  project_id  = var.project_id
  name        = "domain-example-stackit-zone"
  description = "Just an example to manage dns"
  dns_name    = "domain-example.stackit.zone"
}

resource "stackit_dns_record_set" "example" {
  project_id = var.project_id
  zone_id    = stackit_dns_zone.example.zone_id
  name       = "example-a-record.domain-example.stackit.zone."
  type       = "TXT"
  comment    = "My example Service"
  records    = ["foo", "bar"]
}

```
but get the following error after the apply:

```
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to stackit_dns_record_set.example, provider
│ "provider[\"registry.terraform.io/stackitcloud/stackit\"]" produced an unexpected new value:
│ .records[0]: was cty.StringVal("foo"), but now cty.StringVal("bar").
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
╵
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to stackit_dns_record_set.example, provider
│ "provider[\"registry.terraform.io/stackitcloud/stackit\"]" produced an unexpected new value:
│ .records[1]: was cty.StringVal("bar"), but now cty.StringVal("foo").
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
╵
➜  
```

**Comment by vicentepinto98** [comment]
Hi @Slm0n87 , we have encountered a similar issue before and we will work on it soon

**Comment by DiogoFerrao** [comment]
This issue was addressed in Release [v0.15.3](https://github.com/stackitcloud/terraform-provider-stackit/releases/tag/v0.15.3).
If the problem persists, feel free to reach out and reopen this Issue.
